artworks/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
-
+from art_web_app.models import CustomModel
 # Create your models here.
```

Comment:
Keep consistency. You either remove the auto-generated comment (which I recommend) either keep it right above `Categories` class. Anyway, never between your imports (code style - not that it don't work).